javatron/src/main/java/com/newrelic/api/Behaviors/InvalidQueryBehavior.java

```diff
@@ -0,0 +1,29 @@
+package com.newrelic.api.Behaviors;
+
+import java.sql.SQLException;
+
+import com.newrelic.lib.ApplicationContainer;
+import com.newrelic.lib.Inventory.MySQLRepository;
+
+public class InvalidQueryBehavior extends Behavior
+{
+    public static final String Name = "INVALID-QUERY";
+
+    public InvalidQueryBehavior()
+    {
+        this.setName(InvalidQueryBehavior.Name);
```

Comment:
are both lines 10 & 14 needed? 10 in particular. it looks like the same value goes to two places which are effectively doing the same thing -- get/set the name.

Reply:
You know, I'm not actually sure about this one. It's the same model that the other behaviors follow. However, the private value appears to only be used during logging.